.pre-commit-config.yaml

```diff
@@ -52,40 +52,35 @@ repos:
       - id: trailing-whitespace
         exclude: ^.*\.(snap)
         args: ["--markdown-linebreak-ext=md"]
-  - repo: local
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v4.0.0-alpha.8 # Use the sha or tag you want to point at
```

Comment:
We're using 3.3.3 on the repo. I wonder if that'll lead to trouble.

Reply:
Oh wait, I see it uses prettier 3.3.3 below... but this repo is archived and claims it was broken by newer versions of Prettier. Not sure what to make of that yet.

Reply:
Yeah that's the ref o the GHA repo. Looks like the latest -> https://github.com/pre-commit/mirrors-prettier/tags

Reply:
Wasn't sure how much to worry about "This repository was archived by the owner on Apr 11, 2024. It is now read-only." - I guess it might be there forever, but someone could go sweeping up one day.

Reply:
Yeah seems it'd be good hygiene to get rid of this GHA-as-a-submodule and simply `git push` ourselves in our action. I don't think we need an unmaintained reference to a simple GHA here given what we use it for is fairly simple.